app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -54,6 +54,8 @@ def update
         flow_session[:flow_path] ||= 'standard'
       end
 
+      # for the 50/50 state
```

Comment:
nit: perhaps explain what it's doing and why it's here, instead of just why it's here. I _think_ I know, something like this:

```suggestion
      # Mark the FSM verify step completed. This is for the 50/50 state
```